File: src/main/java/org/academy/api/client/gui/command/FillRectDrawCommand.java
package org.academy.api.client.gui.command;

import org.academy.api.client.Render;

import java.util.List;

public class FillRectDrawCommand extends PosColorRectDrawCommand {
    public FillRectDrawCommand(
            float width, float height,
            float red, float green, float blue, float alpha
    ) {
        super(Render.RenderPipelines.POS_COLOR, width, height, red, green, blue, alpha, List.of(), List.of());
    }
}

File: src/main/kotlin/org/academy/api/client/gui/command/FillRectDrawCommand.kt
package org.academy.api.client.gui.command

import org.academy.api.client.Render

class FillRectDrawCommand(
    width: Float, height: Float,
    red: Float, green: Float, blue: Float, alpha: Float
) : PosColorRectDrawCommand(
    Render.RenderPipelines.POS_COLOR,
    width,
    height,
    red,
    green,
    blue,
    alpha,
    listOf(),
    listOf()
)